kaldi/install.bash

```diff
@@ -37,7 +38,7 @@ pip install --user "numpy>=1.15.0"
 # If the directory already exists
 if [ -d "$KALDI_HOME" ]
 then
-    cd "$KALDI_HOME"
+    cd "$KALDI_HOME" || tue-install-error "Missing directory: $KALDI_HOME"
```

Comment:
The if block executes only if the directory exists so having additional is not required